liquibase/maintainer.md

```diff
@@ -0,0 +1 @@
+../.common-templates/maintainer-community.md
```

Comment:
This doesn't seem right -- this will give you a generic value of "the Docker Community" where I think it's much more appropriate if this is Liquibase (since the image _is_ upstream-maintained), maybe something like this:

```markdown
[Liquibase](%%GITHUB-REPO%%)
```

Reply:
Agree